README.md

```diff
@@ -102,6 +102,8 @@ object_client.version.current
 # Returns a struct containing the status.
 # Status includes whether the object is open, assembling, accessioning, or closeable.
 object_client.version.status
+# Get status for a batch of objects
+objects_client.statuses(object_ids: ['druid:bc123df4567', 'druid:bc987gh6543'])
```

Comment:
Consider moving up around line 67 where the other `objects_client` examples are?

Reply:
Done, and left a comment here in README referencing it.

Reply:
I screwed up the commit but snuck it in to main while I was updating the version. Don't tell anyone.